script_vts.py

```diff
@@ -155,74 +155,37 @@ def handle_file():
 ##########################################################
 
 
-def exception_handler(print_exception=False, exception=""):
-    """This function enhances default Python Error handling
-    It will print the line in code that the error occurred on"""
-    if print_exception is True:
-        print(exception)
-        exc_type, exc_obj, exc_tb = sys.exc_info()
-        del exc_type, exc_obj
-        fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
-        print("Exception on line: ", exc_tb.tb_lineno, " in ", fname)
-    return None
-
-
-def verify_at_least_one_command_line_argument(arguments):
+def verify_at_least_x_command_line_arguments(arguments, argument_count):
     """Function checking the number of arguments"""
-    if len(arguments) > 1:
+    if len(arguments) > argument_count:
         return arguments
     return None
 
 
-def formating_file():
+def opening_file_stripping_new_lines():
     """ Checking if the file is well formatted (no blank line, a string on each line),
     if not format the file"""
     try:
         with open(sys.argv[2], encoding="UTF-8") as in_file, open(sys.argv[2], 'r+', encoding="UTF-8") as out_file:
             out_file.writelines(line for line in in_file if line.strip())
             out_file.truncate()
     except IOError as strip_err:
-        exception_handler(True, strip_err)
+        Helpers.exception_handler(True, strip_err)
         exit(1)
     return None
 
 
-def get_path(prompt):
+def get_path_from_user(prompt):
```

Comment:
Let's convert this to a generic/helper function. All it needs is to be passed is an additional string, namely the "Please input the absolute path..."